sync/src/com/kirinpatel/sync.java

```diff
@@ -0,0 +1,80 @@
+package com.kirinpatel;
+
+import com.kirinpatel.net.User;
+import com.kirinpatel.util.UIMessage;
+import jdk.nashorn.api.scripting.URLReader;
+import uk.co.caprica.vlcj.discovery.NativeDiscovery;
+
+import javax.swing.*;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public final class sync {
+
+    private final static int VERSION = 1;
+    private final static int BUILD = 6;
+    private final static int REVISION = 0;
+    public static long deSyncWarningTime = 1000;
+    public static long deSyncTime = 5000;
+    public static boolean showUserTimes = false;
+    public static ArrayList<User> connectedUsers = new ArrayList<>();
+
+    public static void main(String[] args) {
+        if (isUpdated()) {
+            try {
+                UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+            } catch(Exception e) {
+                UIMessage.showErrorDialog(e, "Unable to set look and feel of sync.");
+            } finally {
+                if (verifyDependencies()) {
+                    Launcher.setInstance();

```

Comment:
This has already been implemented, the user will get an error message if dependencies are not verified. However, it will be moved outside of the method.